index.html

```diff
@@ -39,7 +39,7 @@ <h3 class="text-muted">Chromebrew</h3>
       </div>
 
       <div class="row marketing">
-        <div class="col-lg-6">
+        <div class="col-6">
```

Comment:
Thank you.  I couldn't find it by my google search.

Now I understand it is valid, but still not sure what purpose exists.  Why do you choose `col-6` instead of `col-12`?  Thanks.